datashader/glyphs/area.py

```diff
@@ -115,8 +115,8 @@ def extend(aggs, df, vt, bounds, plot_start=True):
             aggs_and_cols = aggs + info(df)
 
             if cudf and isinstance(df, cudf.DataFrame):
-                xs = self.to_gpu_matrix(df, x_name)
-                ys = self.to_gpu_matrix(df, y_name)
+                xs = self.to_cupy_array(df, x_name)
```

Comment:
Is it important to change the name of the method like this? It seems like the name itself wouldn't matter, but maybe this is matching cudf's name in a way that you think will be clearer?

Reply:
the method actually returns a cupy array, (2d if its multiple columns, but a 1d array if its a single column - cudf.Series). Though behind that was, `gpu_matrix` might sound confusing in case it's a Series behind returned. Recent changes to cudf api (https://github.com/rapidsai/cudf/pull/9236), also aim at standardizing the `to_cupy()` and `to_numpy()` methods to avoid any ambiguity.

Reply:
Ok, sounds good! Thanks.